File: web/src/main/java/de/helfenkannjeder/helfomat/service/google/TravelDistanceNotRetrievedException.java
package de.helfenkannjeder.helfomat.service.google;

/**
 * @author Valentin Zickner
 */
public class TravelDistanceNotRetrievedException extends RuntimeException {

    TravelDistanceNotRetrievedException(String message) {
        super(message);
    }

    TravelDistanceNotRetrievedException(Throwable cause) {
        super("Travel distance can not be retrieved.", cause);
    }

}

File: helfomat-api/src/main/kotlin/de/helfenkannjeder/helfomat/api/geopoint/TravelDistanceNotRetrievedException.kt
package de.helfenkannjeder.helfomat.api.geopoint

/**
 * @author Valentin Zickner
 */
class TravelDistanceNotRetrievedException : RuntimeException {
    constructor(message: String?) : super(message)
    constructor(cause: Throwable?) : super("Travel distance can not be retrieved.", cause)
}